### PR TITLE
fvim: Add version 0.2.277

### PR DESCRIPTION
**Pull request by matthewjberger**
This adds a frontend for neovim built with F# + Avalonia. I use this one a lot so I figured I'd add it to scoop to keep it updated more easily

### CODE REVIEW

File: bucket/fvim.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/yatli/fvim/releases/download/v$version-g$matchCommit/fvim-win-x64.zip"
+            }
+        }
+    },
+    "suggest": {
+        "Neovim": "extras/neovim"
```
Comment (Ash258) [inline bucket/fvim.json:31]:
```suggestion
```

File: bucket/fvim.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+    "version": "0.2.273",
+    "description": "Cross platform Neovim front-end UI, built with F# + Avalonia",
+    "homepage": "https://github.com/yatli/fvim",
+    "license": "MIT",
```
Comment (Ash258) [inline bucket/fvim.json:5]:
```suggestion
    "license": "MIT",
    "suggest": {
        "neovim": "extras/neovim"
    },
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Comment by matthewjberger** [comment]
Thank you for reviewing so quickly! All requested changes have been committed. Also, could you please add the `hacktoberfest-accepted` label to this PR once it is accepted?

File: bucket/fvim.json (hunk 2 above)
Comment (matthewjberger) [reply bucket/fvim.json:5]: I've made the 'Neovim' on line 7 all lowercase

**Review by Ash258 — APPROVED** [review]

**Review by rasa — APPROVED** [review]

**Pull request merged**